org.eclipse.m2e.core.ui/src/org/eclipse/m2e/core/ui/internal/wizards/MappingDiscoveryJob.java

```diff
@@ -41,7 +41,7 @@
   private final Collection<IProject> projects;
 
   public MappingDiscoveryJob(Collection<IProject> projects) {
-    super("Discover lifecycle mappings");
+    super("Discover lifecycle mappings"); //$NON-NLS-1$
```

Comment:
This should typically be internationalized; hence `//$NON-NLS-1$` is erroneous here.